helm/templates/primary/replicaset-initiate-configmap.yaml

```diff
@@ -10,17 +10,56 @@ metadata:
     heritage: "{{ .Release.Service }}"
 data:
   replicaset.sh: |
-    #!/bin/sh -x
+    #!/bin/sh
```

Comment:
I have removed the debug flag because the root username and password was being printed in the logs. 
Do we want a flag to enable this? also should I add more `echo` statements for debugging purposes? 
@ravisingal